app/models/menu.py

```diff
@@ -0,0 +1,36 @@
+'''
+Implements the Menu class which stores resturant
+and food item data
+'''
+
+from .resturant import Resturant
```

Comment:
Looks good, just a minor misspelling. 'Resturant' in all instances should be 'Restaurant'

Reply:
good catch! Patching in another branch